webui/src/lib/components/nav.jsx

```diff
@@ -53,12 +56,60 @@ export const Link = (props) => {
     return React.createElement(RouterLink, linkProps, props.children);
 }
 
-export const NavItem = ({ href, active, children }) => {
+export const NavItem = ({href, active, children}) => {
     return (
         <Nav.Item>
             <Link href={href} component={Nav.Link} active={active}>
                 <>{children}</>
             </Link>
         </Nav.Item>
     );
-};
\ No newline at end of file
+};
+
+export function CodeTabPanel({children, value, index, ...other}) {
+    return (
+        <div
+            role="code-tabpanel"
+            hidden={value !== index}
+            id={`code-tabpanel-${index}`}
+            aria-labelledby={`code-tabpanel-${index}`}
+            {...other}
+        >
+            {value === index && (
+                <Box sx={{display: 'flex', flexDirection: 'row', justifyContent: 'space-between', py: 1}}
+                     className={'code-container text-secondary'}>
+                    <Box sx={{ml: 2}}>
+                        {children}
+                    </Box>
+                    <Box sx={{mr: 2}}>
+                        <ClipboardButton icon={<FaRegCopy size={16}/>} variant="link" tooltip="Copy to clipboard"
+                                         text={children} size={'sm'}/>
+                    </Box>
+                </Box>
+            )}
+        </div>
+    );
+}
+
+export const TabsWrapper = ({
```

Comment:
What is the purpose of this component?

Reply:
basically just to set defaults over the `Tabs` component... Might be overkill